01.fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,11 @@
+for (let i = 1; i < 21; i++) {
```

Comment:
```suggestion
for (let i = 1; i <= 20; i++) {
```

とした方が「1から20までの数をプリントする」という日本語とリンクして読みやすいかもしれません。

Reply:
承知いたしました
`i < 21;`と`i <= 20;`のような条件式の場合は、日本語とリンクして読みやすい方を選択します